core/test/unit/services/permissions/can-this_spec.js

```diff
@@ -435,7 +435,10 @@ describe('Permissions', function () {
                         roles: [testUtils.DataGenerator.Content.roles[5]]
                     });
                 });
-                permissions.canThis({api_key_id: 123}) // api key context
+                permissions.canThis({api_key: {
```

Comment:
Am looking into this call at the moment. When changing the type to `content` or anything else, test case didn't fail. Wanna make sure no holes are introduced here :)

Reply:
Did a modification to show that only structure&id matters and not the 'type' in this context. Everything works as it should